config/config.go

```diff
@@ -37,6 +37,9 @@ type serviceConfig struct {
 	MaxConnections int `json:"max_connections" yaml:"max_connections"`
 	// polling interval for checking transfer statuses (seconds)
 	PollInterval int `json:"poll_interval" yaml:"poll_interval"`
+	// name of endpoint with access to local filesystem
+	// (for generating and transferring manifests)
+	Endpoint string `json:"endpoint" yaml:"endpoint"`
```

Comment:
the DRY lover in me wonders if go has ways to automatically set both JSON and YAML to use the same keys

Reply:
I can look into it, but I wouldn't be surprised if not. DRY followed to the hilt can produce a lot of unreadable code that just looks nice and neat when one writes it. Those of us who maintain spend much more time trying to understand it than writing it. Go does get criticized for its boilerplate sometimes, but what I said to Elisha the other day about the ease of porting from Go to other languages has a lot to do with how explicit and regular it is.

Reply:
it depends a bit on the situation -- if the YAML and JSON versions are always going to be identical, _I_ would DRY the code out (if go allowed it); as soon as you start getting exceptions it might be to write things out explicitly.